### PR TITLE
Add PIA (Private Internet Access) support

### PR DESCRIPTION
**Pull request by 0xA50C1A1**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Describe changes:

PIA is a quite popular VPN service.

### CODE REVIEW

File: src/lib/ndpi_main.c [hunk 1]
```diff
@@ -2239,6 +2239,10 @@ static void ndpi_init_protocol_defaults(struct ndpi_detection_module_struct *ndp
 			  "Roughtime", NDPI_PROTOCOL_CATEGORY_SYSTEM_OS,
 			  ndpi_build_default_ports(ports_a, 2002, 0, 0, 0, 0) /* TCP */,
 			  ndpi_build_default_ports(ports_b, 2002, 0, 0, 0, 0) /* UDP */);
+  ndpi_set_proto_defaults(ndpi_str, 0 /* encrypted */, 1 /* app proto */, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_PIA,
+			  "PrivateInternetAccess", NDPI_PROTOCOL_CATEGORY_VPN,
+			  ndpi_build_default_ports(ports_a, 0, 0, 0, 0, 0) /* TCP */,
+			  ndpi_build_default_ports(ports_b, 0, 0, 0, 0, 0) /* UDP */);
```
Comment (IvanNardi) [inline src/lib/ndpi_main.c]: This call is not necessary

**Review by IvanNardi — CHANGES_REQUESTED** [review]
Just a minor issue

**Comment by 0xA50C1A1** [comment]
> Just a minor issue

Ok, I've fixed it.

**Comment by IvanNardi** [comment]
> > Just a minor issue
> 
> Ok, I've fixed it.

Thank you

**Review by IvanNardi — APPROVED** [review]

**Pull request merged**